### PR TITLE
BaSP-TG-96: Implement React Lazy and Suspense

### PR DESCRIPTION
**Pull request by Ansitox**
- Install React Lazy
- Aplly react lazy and suspense

QA:
The navigation between the employees screens should work as they were doing it.

### CODE REVIEW

**Review by renzopimentel — APPROVED** [review]
Everything looks fine

**Review by MaxiG-dev — APPROVED** [review]
Looks good for me! ✅

**Review by melissabonillasellanes — APPROVED** [review]
Everything looks fine!

File: src/routes/employees.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import EmployeeTimeSheets from 'Components/Employees/TimeSheets';
+import EmployeeProfile from 'Components/Employees/Profile';
+
+const Employee = () => {
+  return (
+    <Switch>
+      <Route exact path="/employees" component={Employees} />
+      <Route path="/employees/home" component={EmployeesHome} />
+      <Route path="/employees/timesheets" component={EmployeeTimeSheets} />
+      <Route path="/employees/profile" component={EmployeeProfile} />
```
Comment (matescarabino) [inline src/routes/employees.js:16]: add this line in between

```suggestion
      <Route path="/employees/timesheets/:id" component={EmployeeProfile} />
```

**Review by matescarabino — CHANGES_REQUESTED** [review]

**Review by matescarabino — APPROVED** [review]
LGTM

**Review by NicoSan02 — APPROVED** [review]

**Pull request merged**